tools/cli/commands/docs.js

```diff
@@ -0,0 +1,50 @@
+import child_process from 'child_process';
+import path from 'path';
+
+/**
+ * Command definition for the docs subcommand.
+ *
+ * @param {object} yargs - The Yargs dependency.
+ * @returns {object} Yargs with the docs commands defined.
+ */
+export function docsDefine( yargs ) {
+	yargs.command(
+		'docs [project]',
+		'Parses documentation from a project and outputs them into a JSON file.',
+		yarg => {
+			yarg.positional( 'project', {
+				describe:
+					'Project in the form of type/name, e.g. plugins/jetpack, or type, e.g. plugins, or "all"',
+				type: 'string',
+			} );
+		},
+		async argv => {
+			await docsCli( argv );
+			if ( argv.v ) {
+				console.log( argv );
+			}
+		}
+	);
+
+	return yargs;
+}
+
+/**
+ * Handle args for docs command.
+ *
+ * @param {argv}  argv - the arguments passed.
+ */
+export async function docsCli( argv ) {
+	child_process.spawnSync(
```

Comment:
You may want to do something along the lines of https://github.com/Automattic/jetpack/blob/28c6b4d382d4c0f68df97f2d9b2838ecda42e271/tools/cli/commands/changelog.js#L505-L525 to handle the case where `composer update` needs to be run for the program to work.

Reply:
It'll work, but it has weird behavior in cases where the composer packages are fine but something in the script itself breaks. Stick an `exit(1)` at the end of the runner.php script, for example:
```
$ jetpack docs beta

Extracting PHPDoc from /usr/local/src/automattic/jetpack/projects/plugins/beta.
Data exported to phpdoc.json
Extracting Markdown from readme.md.
Extracting Markdown from docs/quick-start.md.
Extracting Markdown from docs/CONTRIBUTING.md.
Extracting Markdown from docs/CONTRIBUTING.md.
Extracting Markdown from CODE-OF-CONDUCT.md.
Extracting Markdown from docs/monorepo.md.
Extracting Markdown from docs/quick-start.md.
Data exported to markdown.json
Preparing doc-parser package...

Extracting PHPDoc from /usr/local/src/automattic/jetpack/projects/plugins/beta.
Data exported to phpdoc.json
Extracting Markdown from readme.md.
Extracting Markdown from docs/quick-start.md.
Extracting Markdown from docs/CONTRIBUTING.md.
Extracting Markdown from docs/CONTRIBUTING.md.
Extracting Markdown from CODE-OF-CONDUCT.md.
Extracting Markdown from docs/monorepo.md.
Extracting Markdown from docs/quick-start.md.
Data exported to markdown.json
```